### DISCUSSION TITLE
tokopedia.com

### DISCUSSION THREAD

**Discussion by 3xploiton3** [realodix/AdBlockID#1008] [Personal Filter]
<details>
  <summary>Screenshot</summary>
  
  ![Screenshot_20211021-212346048 (1).jpg](https://user-images.githubusercontent.com/19517680/138298192-05a064a4-84df-4b11-817e-58d5d0ee11d3.jpg)

</details>

https://www.tokopedia.com/search?q=crimping

Saya pakai adguard android & via browsee

**Comment by realodix** [comment]
Saya tidak menemukannya Gan, test dengan AdGuard (settingan default + AdBlockID).

Filter di https://github.com/realodix/AdBlockID/issues/999 masih berfungsi dengan baik

<details>
  <summary>Screenshot</summary>
  
  ![noname-crop](https://user-images.githubusercontent.com/1314456/138306375-cf86a065-579e-4bf9-9ee6-4ad7f07601e3.png)
</details>

**Comment by 3xploiton3** [comment]
Maaf ternyata adguard saya belum ke mode https, 

Ini masih ada pop ups (annoyance)


<details>
  <summary>Screenshot</summary>
  
  ![Screenshot_20211022-095114927.jpg](https://user-images.githubusercontent.com/19517680/138385615-30c9d327-4323-477e-91f9-4facfe7e3420.jpg)
  ![Screenshot_20211022-094008470 (1).jpg](https://user-images.githubusercontent.com/19517680/138385636-50b31569-f46b-4f57-8766-01fab8af76a7.jpg)
</details>

**Comment by realodix** [comment]
Sorry Gan @3xploiton3, Saya pribadi menambahkannya ke personal filter dan issue Saya pindahkan ke discussions. 

```
tokopedia.com##[data-testid="WTA-TopSticky"]:has-text(/buka app/i)
tokopedia.com##[data-testid="divSRPPopUpPMPro"]:has(img[src*="/home-search/"])
tokopedia.com##.unf-overlay
tokopedia.com#$#body { margin-top: 0!important; }
```

Agan bisa menambahkannya ke personal filter Agan atau langsung melaporkannya ke filter yang memang khusus menangani annoyance seperti [AdGuard Annoyances](https://github.com/AdguardTeam/AdguardFilters/tree/master/AnnoyancesFilter/sections) atau [Fanboy’s Annoyance](https://github.com/easylist/easylist/tree/master/fanboy-addon).

Alasannya karena dalam konteks ini self promotions (dengan berbagai bentuknya di berbagai website) dalam survei banyak juga user yang tidak masalah karena self-promotion walaupun cukup mengganggu tapi juga mengandung manfaat untuk mereka. Karena filter annoyace jadi satu dengan filter utama, tidak ada pilihan lain untuk mereka selain menonaktifkan AdBlockID secara keseluruhan.

Annoyance yang Saya tangani yang benar-benar tidak ada manfaatnya di sisi user:
- Mouse limitation (gak bisa klik kanan, gak bisa select text, dll)
- Key limitation (gak bisa print screen)
- Fake button
- Popup & Overlay yang disalahgunakan untuk menampilkan iklan
- Safelink yang disalahgunakan untuk menampilkan iklan

Jika ada hal lain untuk personal filter, Agan bisa diskusikan di halaman [discussions](https://github.com/realodix/AdBlockID/discussions).
